OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -169,12 +169,12 @@ public class DefaultSpriteSequence : ISpriteSequence
 		int ISpriteSequence.Stride => stride;
 		int stride;
 
-		[Desc("The amount of directions the unit faces. Use negative values to rotate counter-clockwise.")]
+		[Desc("The amount of directions the unit faces without interpolation. Use negative values to rotate counter-clockwise.")]
 		static readonly SpriteSequenceField<int> Facings = new SpriteSequenceField<int>(nameof(Facings), 1);
 		int ISpriteSequence.Facings => facings;
 		protected int facings;
 
-		[Desc("The amount of directions the unit faces. Use negative values to rotate counter-clockwise.")]
+		[Desc("The amount of directions the unit faces after interpolation. Value must be positive, greater than Facings, and a power of 2.")]
 		static readonly SpriteSequenceField<int> InterpolatedFacings = new SpriteSequenceField<int>(nameof(InterpolatedFacings), 1);
```

Comment:
Can't this be set to `-1` to not use this, and isn't `-1` to default value here?

Reply:
Yes you can use -1 to disable it, but it isn't necessary - you can simply omit the parameter. So it could be rewritten as "If enabled the value must be positive..", but that might be too confusing to people.

Reply:
just mention, "Value defaults to Facings if set to -1"

Reply:
```suggestion
		static readonly SpriteSequenceField<int> InterpolatedFacings = new SpriteSequenceField<int>(nameof(InterpolatedFacings), -1);
```
then as well?

Reply:
The InterpolatedFacings value doesn't actually default to Facings if set to -1, it simply doesn't get used. I could say "This feature is disabled if this parameter is set to -1 or omitted." but worried that might be too long.

Reply:
"No interpolation is used if this value is set to -1"?

Reply:
If I specify that I also need to say it can be omitted, because I don't want people to start putting redundant "InterpolatedFacings: -1" everywhere thinking that the parameter is mandatory when it isn't.

Reply:
the wiki mentions that the default is 1. Can't you change the default to -1?

https://docs.openra.net/en/playtest/sprite-sequences/#defaultspritesequence

Reply:
Could use something like "By default or when set to -1, no interpolation is used."

Reply:
> the wiki mentions that the default is 1. Can't you change the default to -1?

See the code suggestion above.

Reply:
> If I specify that I also need to say it can simply be omitted, because I don't want people to start putting redundant "InterpolatedFacings: -1" everywhere thinking that the parameter is mandatory when it isn't.

it follows the same logic as other traits. There's no need to specify